code/client/src/util.js

```diff
@@ -133,6 +134,21 @@ export default {
       return null
     }
     return parsedOtp
+  },
+
+  getWalletStats: async () => {
+    try {
+      const response = await api.walletStats.getStats()
+      const bi = BigInt(response.totalAmount) / BigInt(10 ** 14)
```

Comment:
The responses should be parsed using bn.js and conversion from wei back and forth is already provided in util or ONEUtil